doc/source/reference/c-api.array.rst

```diff
@@ -204,7 +204,7 @@ From scratch
     new array is *descr*.
 
     If *subtype* is of an array subclass instead of the base
-    :c:data:`&PyArray_Type`, then *obj* is the object to pass to
+    :c:data:`&PyArray_Type<PyArray_Type>`, then *obj* is the object to pass to
```

Comment:
It just wasn't rendered as link: https://docs.scipy.org/doc/numpy/reference/c-api.array.html#c.PyArray_NewFromDescr